binaryclassifier/EmulatorMain.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 23 19:26:17 2016
+
+@author: critter
+
+From https://www.noao.edu/meetings/eventful-universe/Thursday0318/morning/CRTS_TucsonMar10.pdf :
+Discovery rate ~ 1 transient per 10^6 sources detected per epoch So, transient discovery seems very rare. 
+To spice things up, I've made transients appear much more often than what is described above (1 in 50)
+The frequency can be easily changed
+ 
+Each object contains a semi-random number of observations (2-10). Observations come in tuples of (time (seconds), light (no idea)). 
+Objects are arrays of these tuples where the first index is the first observation taken.
+"""
+
+from collections import namedtuple
+import random
+
+class EmulatorMain():
```

Comment:
For class declarations you generally want to use what is called new style class declaration. To use it you just put `object` in the parens here:

```
class EmulatorMain(object):
```